tldw_Server_API/app/main.py

```
@@ -137,6 +141,18 @@ def _claims_rebuild_db_session(
         yield user_id, db_path, db
 
 
+def _run_pg_rls_auto_ensure(backend: Any) -> tuple[bool, bool]:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `backend`

(ANN401)